src/Sia.Gateway/Filters/EventFilters.cs

```diff
@@ -20,27 +28,42 @@ public class EventFilters : Filters<Event>
         public const string KeyValueComparison = "\"{0}\":\"{1}\"";
         public const string KeyComparison = "\"{0}\":";
 
-        public override IQueryable<Event> Filter(IQueryable<Event> source)
+        public bool IsMatchFor(Data.Incidents.Models.Event toCompare)
         {
+<<<<<<< HEAD:src/Sia.Gateway/Filters/EventFilters.cs
+            if (IncidentId.HasValue && toCompare.IncidentId != IncidentId.Value) return false;
+            if (EventTypes != null && EventTypes.Length > 0 && !EventTypes.Contains(toCompare.EventTypeId)) return false;
+            if (Occurred.HasValue && toCompare.Occurred != Occurred.Value) return false;
+            if (EventFired.HasValue && toCompare.EventFired != EventFired.Value) return false;
+
+=======
```

Comment:
Merge conflict... looks like it happened during the merge from master (71ec2b22b7808321ce115f2d1fd99875f6ac24e2)